__main__.py

```
@@ -0,0 +1,600 @@
+    print(f"\n[*] Estimated Total Bounty: ${result.get('total_bounty_estimate', 0):,}")
+    
+    # Save results
+    if args.output:
+        output_path = Path(args.output)
+        output_path.parent.mkdir(parents=True, exist_ok=True)
+        
+        with open(output_path, 'w') as f:
+            json.dump(result, f, indent=2, default=str)
+        print(f"\n[+] Results saved to {output_path}")
+
+def main():
```

Ruff (0.12.2)

`main` is too complex (19 > 10)

(C901)